sklearn/ensemble/_forest.py

```diff
@@ -97,13 +97,13 @@ def _get_n_samples_bootstrap(n_samples, max_samples):
         return n_samples
 
     if isinstance(max_samples, numbers.Integral):
-        if not (1 <= max_samples <= n_samples):
+        if not 1 <= max_samples <= n_samples:
             msg = "`max_samples` must be in range 1 to {} but got value {}"
             raise ValueError(msg.format(n_samples, max_samples))
         return max_samples
 
     if isinstance(max_samples, numbers.Real):
-        if not (0 < max_samples < 1):
+        if not 0 < max_samples < 1:
```

Comment:
I think keeping parenthesis is safer with respect to the binary operations order. Without checking, I am not sure if `not` or `<` will be computed first. So it does not help readability.

Reply:
IMO, Having parenthesis make it easier to read

Reply:
sounds good, done